src/emu/epoc/include/epoc/services/akn/skin/skn.h

```diff
@@ -240,6 +241,25 @@ namespace eka2l1::epoc {
         skn_attrib_info attrib;
     };
 
+    struct skn_layout_info {
+        std::int32_t layout_type;
+        vec2 layout_size;
+    };
+
+    struct skn_scalable_item_def {
+        epoc::pid item_id;
+        std::uint32_t bitmap_handle;
+        std::uint32_t mask_handle;
+        std::int32_t layout_type;
+        bool is_morphing;
+        /**
+         * The original code uses [TTime](http://devlib.symbian.slions.net/belle/GUID-C6E5F800-0637-419E-8FE5-1EBB40E725AA/GUID-A2D3BA72-9B27-3BFF-950A-D7E3B2004821.html)
+         * class, guess they use int64 internally.
+         */
+        std::uint64_t item_timestamp;
```

Comment:
You can make a TTime type in epoc/utils if you like. That would save a lot of time for us in future. Or just keep it like this.

Reply:
do you mean make a class with these helper functions or just a `using ttime = std::uint64_t`?

Reply:
Yeah you can make a helper struct. I don't require it though

Reply:
that would be good, i will do it later :)

Reply:
Thanks a lot!